Contact Section Review

akezeth opened the pull request:
Added Contact Section
Added Footer Section
Integrated 'Formspree' service
Added Appropriate styling in accordance with the Figma template

Gambit142 commented on style.css:
```diff
@@ -409,3 +409,99 @@ h1 {
+  color: #5b5957;
+}
+
+footer {
+  width: 375px;
+  height: 50px;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 48px;
+}
```
- [OPTIONAL] I would advise you to remove the `width` declaration for this element so that elements in the footer can be aligned to the center. Currently, the elements in the footer are aligned to the left.

Gambit142 commented on style.css:
```diff
@@ -409,3 +409,99 @@ h1 {
+  margin-top: 48px;
+}
+
+.socialmedia2 {
+  width: 200px;
+  height: 40px;
+  display: flex;
+  margin-top: 24px;
+  justify-content: space-between;
+}
```
- [ ] Kindly align the social media icons to the center of the page. This would make your design look identical to that of the Figma template.

### Your Design

![Screenshot_6](https://user-images.githubusercontent.com/77106962/228263571-e07c8fa7-1031-4edd-a8c1-44eac5e998e1.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/228263901-7a0a2376-ecae-4a0e-83b3-0a7e08e84bd4.png)

#### Hint: After you remove the `width` declaration from the `footer` element, you can replace the `justify-content` property with the flexbox `gap` property. 👍

Gambit142 commented on index.html:
```diff
@@ -233,5 +233,41 @@ <h2 class="aboutMeTitle">
+                <li><a href="#"><img src="images/medium2.png" alt="social media mediam"></a></li>
+            </ul>
+        </div>
+        <div class="footer-bottomLine">
+           <img src="images/line4.png" alt="blank line image">
+        </div>
```
- [ ] Kindly remove this element. In the design template on Figma, this line appears because it is a prototype that shows how it looks on Apple mobile phones. It isn't part of the design. I would advise that you remove it. 👍

Gambit142 commented on index.html:
```diff
@@ -233,5 +233,41 @@ <h2 class="aboutMeTitle">
+            </form>       
+        </div>
+    </section>
+    <footer>
+        <img src="images/line3.png" alt="line for footer start">
+        <div>
```
- [ ] Kindly make this line occupy the full width of the page. In the design template, the line is from end to end. 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/228265415-5298825b-65ef-4426-847d-9f83a14d1a40.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/228265554-2ecb14b8-ce11-472a-85e4-aafb5c63f967.png)

#### Optional Advice: It would be more semantic to use the `hr` tag since it is a horizontal line

Gambit142 reviewed (changes requested):
Hi @akezeth @Jonathan-kiprono @Nahid1911 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- Use of form element ✔️ 
- Use of HTML validations ✔️ 
- Use of placeholders ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

akezeth commented:
Added CSS flexbox for the form.
Added the missing comma(,) after the 'Inter' value for the font-family property.
Adjusted horizontal alignment issues.
Removed the unnecessary line image at the bottom.
